session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -913,6 +914,37 @@ public SessionDataSet executeQueryStatement(String sql)
         execResp.isIgnoreTimeStamp());
   }
 
+  /**
+   * query eg. select * from paths where time >= startTime and time < endTime
+   * time interval include startTime and exclude endTime
+   * @param paths
+   * @param startTime included
+   * @param endTime excluded
+   * @return
+   * @throws StatementExecutionException
+   * @throws IoTDBConnectionException
+   */
+
+  public SessionDataSet executeRawDataQuery(List<String> paths, long startTime, long endTime)
+          throws StatementExecutionException, IoTDBConnectionException {
+
+    TSRawDataQueryReq execReq = new TSRawDataQueryReq(sessionId, paths, startTime, endTime);
+    execReq.setFetchSize(fetchSize);
+
+    TSExecuteStatementResp execResp;
+    try {
+      execResp = client.executeRawDataQuery(execReq);
+    } catch (TException e) {
+      throw new IoTDBConnectionException(e);
+    }
+
+    RpcUtils.verifySuccess(execResp.getStatus());
+    return new SessionDataSet("", execResp.getColumns(), execResp.getDataTypeList(),
```

Comment:
It seems that the statement(sql) in SessionDataSet is not needed. We could remove the statement field in TSFetchResultsReq in rpc.thrift.

Reply:
> It seems that the statement(sql) in SessionDataSet is not needed. We could remove the statement field in TSFetchResultsReq in rpc.thrift.

we should not remove the field, because TSFetchResults has the field in the previous release 0.10. User may use old client to query data from new release, we should keep our system backward compatible.

Reply:
It's ok, but it seems that the client-0.10 can not connect to server-0.11 already...